<launch>

    <node name="audio_playback" type="alsa_playback_node" pkg="audio_utils" output="screen" respawn="true">
        <param name="device"   type="string"   value="default"/>
        <param name="format"   type="string"   value="signed_16"/>
        <param name="channel_count"   type="int"   value="1"/>
        <param name="sampling_frequency" type="int" value="44100"/>
        <param name="frame_sample_count" type="int" value="2048"/>
        <param name="latency_us" type="int" value="4000" />
        <remap from="audio_in" to="/audio_signed_16_44100"/>
    </node>

    <node name="image_uncompress" pkg="image_transport" type="republish" output="screen" respawn="true"
        args="compressed in:=/camera/color/image_raw raw out:=/camera/color/image_raw"/>

    <name="rqt_image_view" type="rqt_image_view" pkg="rqt_image_view" args="/camera/color/image_raw" output="screen" respawn="true"/>

</launch>
